packages/core/doc-links/core-doc-links-browser/index.ts

```diff
@@ -7,3 +7,9 @@
  */
 
 export type { DocLinksStart } from './src/types';
+
+export {
+  GITHUB_CREATE_ISSUE_LINK,
+  KIBANA_ASK_ELASTIC_LINK,
+  KIBANA_FEEDBACK_LINK,
+} from './src/constants';
```

Comment:
I don't think this is what @lukeelmers meant. At least that wasn't what I had in mind :) 

The idea was to add those links  to the service's contract, here: https://github.com/elastic/kibana/blob/4d812c1d24c74e3dfb09f42a1953de8e17fb969c/packages/kbn-doc-links/src/types.ts#L21 and here https://github.com/elastic/kibana/blob/4d812c1d24c74e3dfb09f42a1953de8e17fb969c/packages/kbn-doc-links/src/get_doc_links.ts#L17

So that consumers can access them as they would access any other docLink.

Reply:
Good point. 👍  Addressed in https://github.com/elastic/kibana/pull/160535/commits/1daaa30610e0ddc7a5c43a58c8c8402ad939f72a